infrastructure/helm/quantumserverless/templates/pvcs.yaml

```diff
@@ -1,28 +1,21 @@
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: gateway-volume
-  labels:
-    type: local
-  annotations:
-    pv.beta.kubernetes.io/gid: "1001"
-spec:
-  storageClassName: manual
-  capacity:
-    storage: 2Gi
-  accessModes:
-    - ReadWriteMany
-  hostPath:
-    path: "/mnt/data/tmp"
----
-apiVersion: v1
+{{ if .Values.cosEnable }}
 kind: PersistentVolumeClaim
+apiVersion: v1
 metadata:
   name: gateway-claim
+  annotations:
+    ibm.io/auto-create-bucket: "false"
+    ibm.io/auto-delete-bucket: "false"
+    ibm.io/bucket: {{ .Values.cos.bucket | quote }}
+    ibm.io/quota-limit: "false"
+    ibm.io/endpoint: {{ .Values.cos.endpoint | quote }}
+    ibm.io/tls-cipher-suite: "default"
+    ibm.io/secret-name: {{ .Values.cos.secretName | quote }}
 spec:
-  storageClassName: manual
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
   resources:
     requests:
-      storage: 1Gi
+      storage: 10Gi
```

Comment:
Let's move this as variable too, for future deployment we will need more probably :)

Reply:
Solved in: [90e5295](https://github.com/Qiskit-Extensions/quantum-serverless/pull/668/commits/90e52951b4cc79aa90d40a72f365f28771e28dff), [92f3d6e](https://github.com/Qiskit-Extensions/quantum-serverless/pull/668/commits/92f3d6efc583c4018136a9d949929d605435a4df)